src/main.js

```diff
@@ -12,9 +12,6 @@ import { getRequestToken } from '@nextcloud/auth'
 import AdminSettings from './AdminSettings.vue'
 import store from './store/index.js'
 
-// Styles
-import '@nextcloud/dialogs/dist/index.css'
```

Comment:
While server provides some, we would be required to use exact the same version, so instead keep it.

Should look like: `import '@nextcloud/dialogs/style.css'`

Reply:
the library dropped this file and compiling fails if the line is kept. See note in the library:
> If you using @nextcloud/dialogs >= 4.0 you don't need any svg or scss loader in you projects anymore.

Reply:
Oh wait, just the loader? I read it all wrong?

Reply:
Yes because instead of importing it you could also just provide that CSS file.

But for apps I recommend still importing it (it is called `style.css` since 5.0.0), and let your bundler decide how to handle.